ts/Stock/Navigator/StandaloneNavigator.ts

```diff
@@ -0,0 +1,330 @@
+ * The standalone navigator options with chart-like structure.
+ */
+class StandaloneNavigator {
+
+    public eventsToUnbind: Array<Function> = [];
+    public navigator: Navigator;
+    public boundAxes: Array<Axis> = [];
+    public options: StandaloneNavigatorOptions;
+    public userOptions: DeepPartial<StandaloneNavigatorOptions>;
+
+    /* *
+     *
+     *  Static Functions
+     *
+     * */
+
+    /**
+     * Factory function for standalone navigator.
+     *
+     * @function Highcharts.navigator
+     *
+     * @param {string|Highcharts.HTMLDOMElement} [renderTo]
+     * The DOM element to render to, or its id.
+     *
+     * @param {StandaloneNavigatorOptions} options
+     * The standalone navigator options with chart-like structure.
+     *
+     * Returns the navigator object.
+     */
+    public static navigator(
+        renderTo: (string|globalThis.HTMLElement),
+        options: DeepPartial<StandaloneNavigatorOptions>
+    ): StandaloneNavigator {
+        const nav = new StandaloneNavigator(renderTo, options);
+
+        if (!G.navigators) {
+            G.navigators = [nav];
+        } else {
+            G.navigators.push(nav);
+        }
+
+        return nav;
+    }
+
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+    constructor(
+        element: (string | globalThis.HTMLElement),
+        userOptions: DeepPartial<StandaloneNavigatorOptions>
+    ) {
+        this.userOptions = userOptions;
+        this.options = merge(
+            (G as any).getOptions(),
+            StandaloneNavigatorDefaults,
+            userOptions
+        ) as StandaloneNavigatorOptions;
+
+        const chart = new Chart(element, this.options);
+
+        this.navigator = new Navigator(chart);
+        chart.navigator = this.navigator;
+        this.initNavigator();
+    }
+
+    /**
+     * Binds an axis to the standalone navigator,
+     * allowing the navigator to control the range.
+     *
+     * @param {Axis | Chart} axisOrChart
+     *        The Axis or Chart to bind to the navigator.
+     */
+    public bind(axisOrChart: Axis | Chart): void {
+        const nav = this;
+        // If the chart is passed, bind the first xAxis
+        const axis = (axisOrChart instanceof Chart) ?
+            axisOrChart.xAxis[0] :
+            axisOrChart;
+
+        if (!(axis instanceof Axis)) {
+            return;
+        }
+
+        const { min, max } = this.navigator.xAxis;
+
+        this.boundAxes.push(axis);
+
+        // Show axis' series in navigator based on showInNavigator property
+        axis.series.forEach((series): void => {
+            if (series.options.showInNavigator) {
+                nav.addSeries(series.options);
+            }
+        });
+
+        // Set extremes to match the navigator's extremes
+        axis.setExtremes(min, max);
+    }
+
+    /**
+     * Unbinds an axis from the standalone navigator.
+     *
+     * @param {Chart | Axis | undefined} axisOrChart
```

Comment:
We need to describe all three params. Eg we need to add info `undefined` removes all charts and axes from nav